miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -365,32 +333,28 @@ end
 #! Output: [offset_slot_index]
 export.apply_storage_offset
     # verify that slot_index is in bounds
-    dup.1 push.MAX_STORAGE_SLOT_INDEX lt assert.err=ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS
+    dup.1 exec.memory::get_num_storage_slots lt assert.err=ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS
     # => [storage_offset, slot_index]
 
     # verify that the computed offset_slot_index is in bounds
-    add dup push.MAX_STORAGE_SLOT_INDEX lt assert.err=ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS
+    add dup exec.memory::get_num_storage_slots lt assert.err=ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS
     # => [offset_slot_index]
 end
 
-#! Gets an item from the account storage. Panics if the index is out of bounds.
+#! Gets an item from the account storage
 #!
 #! Stack: [index]
 #! Output: [VALUE]
 #!
 #! - index is the index of the item to get.
 #! - VALUE is the value of the item.
 export.get_item
```

Comment:
Here and in other similar places, I would add a comment that we assume that `index` has already been validated and is within bounds.

Reply:
Added: 

```text
#! Note:
#!    - We assume that index has been validated and is within bounds
#!
```